packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.spec.js

```diff
@@ -8,37 +8,36 @@ describe('ZooniverseLogo', function () {
   const ID = 'foobar'
   const SIZE = '100px'
 
-  before(function () {
-    wrapper = mount(<ZooniverseLogo id={ID} size={SIZE} />)
+  describe('with standard props', function () {
+    beforeEach(function () {
+      render(
+        <ZooniverseLogo id={ID} size={SIZE} />
+      )
+    })
+
+    it('should render without crashing', function () {
+      expect(screen).to.be.ok()
+    })
+
+    it('should set the height and width from the `size` prop', function () {
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
```

Comment:
I'm slightly baffled as to why this is rendering "Zooniverse Logo" instead of "ZooniverseLogo.title" in the tests, but it seems to be a difference in how the lib-classifier's components/tests are set up vs lib-react-components's components/tests. (It's that `import '../translations/i18n'` bit setting up the fallback language, innit?)

Reply:
#3761 might change this.